library/pkcs11_client.c

```diff
@@ -239,7 +239,7 @@ static int pkcs11_verify( void *ctx_arg,
     mbedtls_pk_pkcs11_context_t *ctx = ctx_arg;
     CK_RV rv;
     CK_MECHANISM mechanism = {0, NULL_PTR, 0};
-    unsigned char *decoded_sig = NULL_PTR;
+    unsigned char *decoded_sig = NULL;
```

Comment:
Suggestion: I think a stack buffer of static size `2 * MBEDTLS_ECP_MAX_BYTES` would be fine here.